README.md

```diff
@@ -68,17 +68,41 @@ remove *a single* docker image <br>
 remove *all* containers and images <br>
 `docker system prune -a` <br>
 
-### Sending Emails <br>
-Create the file ".env" in the same directory as "send_email.py" and format <br>
-the file as following: <br>
+### Running the Application <br>
+
+To begin, OpenCV will need to be installed on the Raspberry Pi to make use of the motion detection feature. Run the following commands to enable and run this. *Note: This process will take around 2 hours to complete. *
+
+```
+chmod +x OpenCV-4-5-5.sh
+./OpenCV-4-5-5.sh
+```
+
+Once OpenCV is installed, run the following commands to enable access to the startup script and run it.
+
 ```
-EMAIL_HOST=smtp.gmail.com
-EMAIL_HOST_USER=your_email_address@gmail.com
-EMAIL_HOST_PASSWORD=secret_key
-RECIPIENT_ADDRESS=your_email_address@gmail.com
+chmod +x run.sh
+./run.sh
 ```
 
-To generate the secret key follow these steps:
+The ```./run.sh``` will be used to start the application every time.
+
+### Stopping the Application <br>
+
+Hold Ctrl-C to stop the dashboard. Next type the following command to list the current running processes:
+
+```
+ps
+```
+
+From here type the following command followed by the process listed as "Python3". This will stop the the motion detection script from running.
+
+```
+kill <PID Number>
+```
+
+### Sending Emails <br>
+
+While running this script, it will prompt if you want to configure using your email to send notifications. To enable this feature, it will ask for a secrete key to have access to send from your email address. To generate the secret key follow these steps:
```

Comment:
I think this is fine for now, however, I think we would like to just have a form on the web app to get the email address and secret key and store those in the database, which can then be grabbed by the send email script.